user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Changing_the_IP_of_a_DMA.md

```diff
@@ -204,7 +204,7 @@ For a Failover DMA within a cluster that does not use the Cassandra cluster feat
 
    1. Close the SLNetClientTest tool.
 
-## DMA in a DMS using a Cassandra cluster
+## DMA in a DMS using a Cassandra cluster locally
```

Comment:
This might not be clear enough. Do you mean that this only applies when Cassandra is hosted on the same machine as DataMiner?

Reply:
Yes, this is only applicable when the DB is locally

Reply:
That means there's important information missing on this page, i.e. how you can change a DMA's IP if you use the recommended setup with a Cassandra Cluster on Linux. Is this something you could also add, or is there someone in your team who might have time for this?

Reply:
I've added a clarification in the paragraph below. I'd like to merge this update already. Would it be possible to add a task (if this hasn't been done yet) to add the missing information mentioned above later?